### PR TITLE
Add Kyma integration test scripts for the Prow job

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Adds Kyma deployment and test script to be used for Kyma integration job on Prow
- Adds installation script of the necessary tooling for Debian to deploy Kyma on Minikube

**Related issue(s)**

See [#60](https://github.com/kyma-project/test-infra/issues/60)

### CODE REVIEW

File: prow/scripts/deploy_and_test_kyma.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (aszecowka) [inline prow/scripts/deploy_and_test_kyma.sh]: in prow directory, our scripts use "-" and "_" as a separator, please make it consistent.

File: prow/scripts/deploy_and_test_kyma.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+###
```
Comment (aszecowka) [inline prow/scripts/deploy_and_test_kyma.sh]: I am a little bit afraid about mixing all prow scripts in one directory. Maybe we should create a separate directory for running kyma-int on vm, and call it "kyma-integration-on-debian" ?

File: prow/scripts/install_deps_debian.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+sudo apt-get update
+sudo apt-get install -y \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     gnupg2 \
+     socat \
+     software-properties-common
+
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
```
Comment (michal-hudy) [inline prow/scripts/install_deps_debian.sh]: Instead of installing it every time, maybe we should prepare a custom VM image?

Comment (suleymanakbas91) [reply prow/scripts/install_deps_debian.sh]: We already have an issue to investigate this [here](https://github.com/kyma-project/test-infra/issues/47). So, this shouldn't be a blocker for this PR.

**Pull request merged**